packages/get_state_manager/lib/src/simple/get_view.dart

```diff
@@ -21,9 +24,9 @@ import 'package:get_instance/get_instance.dart';
 /// }
 ///``
 abstract class GetView<T> extends StatelessWidget {
-  const GetView({Key key}) : super(key: key);
+  const GetView({Key key, this.tag}) : super(key: key);
 
-  final String tag = null;
+  final String tag;
```

Comment:
This requires the implementation of the tag, since it is an abstract class. It is strange that it receives null, but unfortunately dart is still confused between implementation and extends. This should not happen, but as it does, it should be maintained

Reply:
done....